### PR TITLE
Remove unicode content from CONTRIBUTORS file.

### PR DESCRIPTION
**Pull request by leplatrem**
Tests fail with python3 because of this : 
https://travis-ci.org/mozilla-services/cliquet/jobs/62362839

In setup.py it should be read like this in python3:

    f = open('CONTRIBUTORS.txt', encoding='utf-8')

and in python 2:

    f = codecs.open('CONTRIBUTORS.txt', encoding='utf-8')

...so instead of introducing compat code in setup, just
remove utf8 from CONTRIBUTORS file.

@tarekziade r?

### CODE REVIEW

**Comment by Natim** [comment]
r-

**Comment by ametaireau** [comment]
This doc is missing @michielbdejong as well! (Michiel de Jong)

I would update the setup.py to use unicode rather than updating this doc to remove them.

**Comment by leplatrem** [comment]
Please do it in this branch :)

I wonder why this didn't popup earlier !

**Comment by Natim** [comment]
The problem will come back as soon as we have a Russian or Chinese contributor.

**Comment by leplatrem** [comment]
We can also refuse to merge pull-requests from authors that have non ascii names !

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@ Contributors
 ============
 
 * Alexis Metaireau <alexis@mozilla.com>
+* Michiel de Jong <michiel@unhosted.org>
```
Comment (ametaireau) [inline CONTRIBUTORS.rst]: I believe it was sorted. Gah, doesn't matter.

Comment (Natim) [inline CONTRIBUTORS.rst]: I was sorting them by the lastname didn't see that it changed.

Comment (Natim) [inline CONTRIBUTORS.rst]: Fixed with https://github.com/mozilla-services/cliquet/commit/54b34337e83806ecb8b4448b66642d6518dcb1e0

**Comment by michielbdejong** [comment]
Thanks for the mention! :)

**Pull request merged**